kinematic_properties.py

```diff
@@ -132,7 +132,7 @@ def get_vmax(mass, radius):
 def get_inertia_tensor(mass, position):
 
     # 3x3 inertia tensor
-    Itensor = (mass[:, None, None] * position[:, None:, None] * position[:, None]).sum(
+    Itensor = (mass[:, None, None]/mass.sum() * position[:, None:, None] * position[:, None]).sum(
```

Comment:
Why are you changing the units of the inertia tensor?Do you know of anyone else who uses this property that will need to be informed of the change?